pkgs/servers/pufferpanel/default.nix

```diff
@@ -1,43 +1,59 @@
 { lib
-, buildGoModule
 , fetchFromGitHub
-, makeWrapper
-, fetchzip
 , fetchpatch
-, pathDeps ? [ ]
+, applyPatches
+, buildGoModule
+, pkgsBuildBuild # buildNpmPackage
+, makeWrapper
+, go-swag
 , nixosTests
+, pathDeps ? [ ]
```

Comment:
Why do we need this option? Rebuilding a package to change the PATH env in the wrapper is rather wasteful. People can just add the package to the environment.

Reply:
This was added in #128243 when package was introduced, not sure why though. If this won’t be considered a breaking change for backport, I’m all in for removing it.

Reply:
Should be alright in my opinion. Changing how the frontent is build as a small chance for breakage anyway.

Reply:
Done.